NWN.FinalFantasy.CLI/HakBuilder.cs

```diff
@@ -121,6 +183,15 @@ private void CompileHakpak(string hakName, string folderPath)
 
                 process.WaitForExit();
             }
+
+            if (_checksumDictionary.TryGetValue(hakName, out var checksum))
```

Comment:
You can simplify this if statement a bit. Make it 

if (!_checksumDictionary.TryGetValue(hakName, out var checksum))
{
    checksum = ChecksumUtil.ChecksumFolder(folderPath);
}

and you're good.